stress/shell/src/main/java/alluxio/stress/cli/GetPinnedFileIdsBench.java

```diff
@@ -163,12 +165,12 @@ private RpcTaskResult runRPC() throws Exception {
         pointStopwatch.reset();
         pointStopwatch.start();
 
-        Set<Long> pinned = client.getPinList();
+        int numPinnedFiles = client.getPinListLength();
 
         pointStopwatch.stop();
 
-        if (pinned.size() != mParameters.mNumFiles) {
-          result.addError(String.format("Unexpected number of files: %d", pinned.size()));
+        if (numPinnedFiles != mParameters.mNumFiles) {
+          result.addError(String.format("Unexpected number of files: %d", numPinnedFiles));
```

Comment:
When A mismatches with B, better print both A and B

Reply:
6fb2ad27114d5689b74afbc20e9e325ebecd9322